### DISCUSSION TITLE
How to use the title from the BrowserWindow options

### DISCUSSION THREAD

**Discussion by jsheffers** [cawa-93/vite-electron-builder#501] [Q&A] [answered]
Setting the **title** for the `BrowserWindow` at the time of creation does not have any effect when the app actually starts.

<img width="549" alt="CleanShot 2021-10-13 at 11 52 02@2x" src="https://user-images.githubusercontent.com/656483/137169274-f90d0664-441d-4f08-8e76-0b43104504ea.png">

The app still loads with w/e title is defined in index.html of the renderer. In this case I would assume the title should be "Something custom" instead it's still "Vite App".

<img width="1413" alt="CleanShot 2021-10-13 at 11 52 37@2x" src="https://user-images.githubusercontent.com/656483/137169414-e022b431-cdcd-48b0-8efc-fb283d5a396d.png">

Has anyone achieved this?

**Comment by jsheffers** [comment, marked as answer]
This works:  https://github.com/IndexXuan/vite-plugin-html-template

**Comment by cawa-93** [comment]
From Electron doc:

> `title` String (optional) -  Default window title. Default is "Electron". **If the HTML tag <title> is defined in the HTML file loaded by loadURL(), this property will be ignored**.
